_gsocblogs/2022/blog_ROOT_Neelshah.md

```diff
@@ -0,0 +1,366 @@
+---
+project: ROOT
+title: Inference of Deep Learning models in TMVA/SOFIE
+author: Neel Shah
+photo: blog_authors/NeelShah.jpg 
+date: 24.07.2022 
+year: 2022 
+layout: blog_post 
+logo: ROOT-logo.png 
+intro: |
+Mid Evaluation Report of GSOC 2022 for the project :- Inference of Deep Learning Models in TMVA/SOFIE.
+---
+
+# Mid Evaluation Report for GSOC 2022
+
+<div align="center">
+<img src="https://user-images.githubusercontent.com/84740927/170821829-f631f5f7-c410-429a-acdc-d9cc81d13dd3.jpeg" alt="drawing" width="500"/>
+</div> 
+
+#### Project :- ROOT - TMVA SOFIE Developments - Inference Code Generation for Deep Learning models
```

Comment:
Replace all `:-` with `:` in the body text

Reply:
Done.